src/SFA.DAS.ApprenticeAccounts.Api.AcceptanceTests/WorkflowTests/ApiFixture.cs

```diff
@@ -1,18 +1,19 @@
+
+
+#nullable enable
+
 using AutoFixture;
 using FluentAssertions;
 using Microsoft.AspNetCore.JsonPatch;
 using NUnit.Framework;
 using SFA.DAS.ApprenticeAccounts.Application.Commands.CreateApprenticeAccountCommand;
 using SFA.DAS.ApprenticeAccounts.Data.Models;
-using SFA.DAS.ApprenticeAccounts.DTOs;
+using SFA.DAS.ApprenticeAccounts.DTOs.Apprentice;
```

Comment:
Putting the comment here for visibility. Accounts Api had acceptance tests over unit tests. It's great you've added a unit test project but may want to see if you can add an acceptance test as well for completeness.

Reply:
Would like to do this - I have never written acceptance tests before so need some help getting started